### ISSUE TITLE
PLZ HAVE A LOOK!!! Is there a use of clip-retrieval during fine-tuning of faces?

### ISSUE THREAD

**Issue by SKDDJ** [adobe-research/custom-diffusion#72]
Hello,

I'm interested in your project and have a question regarding the use of clip-retrieval during fine-tuning of faces. 

In the project's README, you mentioned that all the images in your paper were generated without using clip-retrieval because the results were similar. However, I was wondering if you used clip-retrieval during the fine-tuning process for faces. Could you please clarify this for me?

Thank you!

**Comment by SKDDJ** [comment]
I used 10 pictures of a person's face to fine-tune the model, but the results don't seem to be as good as described in the README.md. Can you provide more details on how to train the model?

**Comment by nupurkmr9** [comment]
Hi, thanks for your interest in this work. 
You can find more detailed instructions for fine-tuning on faces in the issue [here](https://github.com/adobe-research/custom-diffusion/issues/13#issuecomment-1434066085). These results are with regularization images obtained using clip-retrieval only. 
Training on faces is better with fine-tuning all cross-attention parameters and requires longer training time with more images compared to other categories. 
Hopefully, this should help in getting better results.

**Comment by SKDDJ** [comment]
@nupurkmr9 ,Thank you for your prompt response and for sharing the detailed instructions for fine-tuning on faces. I appreciate your help.

I'm happy to inform you that I was able to deal with the issue. I followed your instructions and fine-tuned all cross-attention parameters, which required longer training time with more images compared to other categories, as you mentioned. The results are much better now, and I'm satisfied with the performance.

Thanks again for your support:)
